client/components/LearnerViewDirective.js

```diff
@@ -21,104 +21,172 @@ tie.directive('learnerView', [function() {
+                  <hr>
+                </div>
+              </div>
+              <div id="tie-instructions" class="tie-instructions">
+                <p ng-repeat="paragraph in instructions">
+                  {{paragraph}}
+                </p>
+              </div>
+            </div>
+            <button ng-click="showNextPrompt()" ng-if="nextButtonIsShown"
+                id="next-button">Next</button>
+            </button>
+          </div>
+          <div id="coding-ui">
+            <div id="feedback-window">
+              <div id="tie-feedback" class="tie-feedback">
+                <p ng-repeat="paragraph in feedbackMessages track by $index"
+                    class="tie-feedback-paragraph">
+                  {{paragraph}}
+                </p>
+              </div>
+            </div>
+            <div id="coding-window">
+              <div id="lang-terminal">
+                <!--textarea name="textarea" id="coding-terminal"
+                    class="terminal">
+                </textarea-->
+                <div id="coding-terminal" class="terminal">
+                  <ui-codemirror ui-codemirror="codeMirrorOptions" ng-model="code" class="codemirror-container"></ui-codemirror>
+                </div>
+                <select id="lang-select-menu" name="lang-select-menu">
+                  <option value="Python" selected>Python</option>
+                  <option value="JavaScript">JavaScript</option>
+                </select>
+                <button id="run-button" class="run-button"
+                    ng-class="{'active': !nextButtonIsShown}"
+                    ng-click="submitCode(code)"
+                    ng-disabled="nextButtonIsShown">
+                  Run
+                </button>
+              </div>
+            </div>
           </div>
-        </div>
-        <div class="tie-instructions">
-          <p ng-repeat="paragraph in instructions">
-            {{paragraph}}
-          </p>
-        </div>
-
-        <button ng-if="nextButtonIsShown" class="tie-action-button active"
-                ng-click="showNextPrompt()">
-          Next
-        </button>
-      </div>
-
-      <div class="tie-learner-view-right-column">
-        <div class="tie-coding-window">
-          <ui-codemirror ui-codemirror="codeMirrorOptions" ng-model="code"></ui-codemirror>
-        </div>
-
-        <button type="button" class="tie-action-button"
-                ng-class="{'active': !nextButtonIsShown}"
-                ng-click="submitCode(code)"
-                ng-disabled="nextButtonIsShown">
-          Run
-        </button>
-
-        <div class="tie-feedback">
-          <p ng-repeat="paragraph in feedbackMessages track by $index"
-             class="tie-feedback-paragraph">
-            {{paragraph}}
-          </p>
         </div>
       </div>
-
       <style>
-        .tie-learner-view-left-column {
+        body {
+          font-family: Roboto, 'Helvetica Neue', 'Lucida Grande', sans-serif;
```

Comment:
Could we, as a general rule, alphabetize CSS (so, background-color, then font-family, then font-size)? It makes it easier to find whether a key is specified, especially in rules that have many keys.